src/include/migraphx/compile_options.hpp

```diff
@@ -40,6 +40,12 @@ struct compile_options
 
     bool fast_math       = true;
     bool exhaustive_tune = false;
+
+    /**
+     * Converts floating point values to double on the ref target.
+     * Currently only used by the driver.
+     */
+    bool fp_to_double = false;
```

Comment:
`compile_options` are for the `compile` function. If its not being used by `compile` then it shouldn't be there.

Reply:
The option is used to enable compiler passes, so it makes sense as a compile_option like disable_fast_math.

Reply:
But we ignore the option for the GPU/CPU targets.